Gruntfile.js

```diff
@@ -62,14 +62,16 @@ module.exports = function(grunt) {
 
         watch: {
             files: ['js/**/*.js'],
-            tasks: ['lint', 'concat', 'uglify']
+            tasks: ['concurrent:lint', 'concurrent:concat', 'concurrent:uglify']
+        },
+
+        concurrent: {
+            lint: ['jshint', 'jscs'],
+            concat: ['concat'],
+            uglify: ['uglify']
         }
     });
 
-    grunt.registerTask('lint', [
-        'jshint',
-        'jscs'
-    ]);
+    grunt.registerTask('default', ['concurrent:lint', 'concurrent:concat', 'concurrent:uglify', 'watch']);
```

Comment:
The default task usually build the project. This means it doesn't need the watch task. It should use a separated alias like "dev". Also, because you're calling all the "concurrent" tasks,  you should be able to just declare:

```
grunt.registerTask('default', ['concurrent']);
```